README.md

```diff
@@ -27,7 +27,7 @@ cd jerryscript
 
 ### Building JerryScript
 ```bash
-make release.linux -j
+python tools/build.py
```

Comment:
Thanks for the notice, I missed it earlier, so I added the parallel build by default in the build script.